app/analysis/edit/controller.js

```diff
@@ -30,7 +31,10 @@ export const AnalysisEditQueryParams = AnalysisQueryParams.extend({
 const prepareParams = params =>
   cleanParams(
     underscoreQueryParams(
-      serializeParachuteQueryParams(params, AnalysisEditQueryParams)
+      serializeParachuteQueryParams(
```

Comment:
Damn, this level of nesting would almost justify pulling in an FP lib to get `pipe` support ;)

Reply:
Blargh *library*.. Waiting for this to be supported http://tc39.github.io/proposal-pipeline-operator/..

Reply:
right - as discussed